app/browser/api/ledger.js

```diff
@@ -182,16 +182,17 @@ const paymentPresent = (state, tabId, present) => {
   } else {
     delete ledgerPaymentsPresent[tabId]
   }
-
-  if (getSetting(settings.PAYMENTS_ENABLED) && present) {
-    if (!balanceTimeoutId) {
-      module.exports.getBalance(state)
-    }
-
+  if (present) {
     appActions.onPromotionGet()
+    clearTimeout(togglePromotionTimeoutId)
```

Comment:
*nit*

Let's wrap this in an `if` statement (`if (togglePromotionTimeoutId)`)

Reply:
Oh! Not a nit for me. Thanks.